index.html

```diff
@@ -97,7 +97,7 @@ <h3>
                         <span class="carrot"></span>
                         <div class="device-box-header">Oculus Rift</div>
                         <span>
-                            Works with <a href="https://webvr.rocks/firefox">Firefox Nightly</a> and
+                            Works with <a href="https://webvr.rocks/firefox">Firefox</a> and
                             <a href="https://www.supermedium.com">Supermedium</a>
                             on Windows
```

Comment:
nit: there should be a period `.` here

Reply:
and probably elsewhere too - for consistency:

![image](https://user-images.githubusercontent.com/203725/38218172-b9bd55ec-3685-11e8-9a49-5ad5ba738c3c.png)

Reply:
Done